recipes/azure-identity-cpp/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eux
+
+# Disable installing dependencies with vcpkg
+# https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#customize-the-vcpkg-dependency-integration
+# https://github.com/search?utf8=%E2%9C%93&q=AZURE_SDK_DISABLE_AUTO_VCPKG&type=code
+export AZURE_SDK_DISABLE_AUTO_VCPKG=ON
+
+# https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+if [[ "${target_platform}" == osx-* ]]
+then
+  CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+# For some reason CMAKE_BUILD_TYPE=Release isn't included in CMAKE_ARGS by
+# ctng-compiler-activation-feedstock, even though it is for vc-feedstock and
+# clang-compiler-activation-feedstock
+if [[ "${target_platform}" == linux-* ]]
+then
+  CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release"
+fi
```

Comment:
Just add `-DCMAKE_BUILD_TYPE=Release` to the CMake invocation, the duplication doesn't hurt. Having extra treatment should be reserved for things where that's not possible or reasonable for some reason. Also feel free to raise an issue about this on the ctng activation feedstock.

Reply:
I had already raised the issue: https://github.com/conda-forge/ctng-compiler-activation-feedstock/issues/94

Reply:
For potential future readers of this thread, the ctng-compiler-activation-feedstock now includes `CMAKE_BUILD_TYPE=Release` in the exported env var `CMAKE_ARGS` (https://github.com/conda-forge/ctng-compiler-activation-feedstock/pull/95/files)